.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,5 +1,10 @@
 ---
 name: Bug Report
+about: Problems, deficiencies, inaccuracies or crashes discovered on Solidity.
```

Comment:
One more thing I just remembered. Are you sure these will even be picked up? Currently the descriptions that are displayed come from `config.yml`. And not having them in there  was causing weird issues with links specified in `config.yml` not being picked up tye last time I tried: #12419.

Reply:
Unlikely, since that was over 2 yo.